### PR TITLE
Add guide to install packages from the bintray apt repo

### LINKED ISSUE #248: Add guide to install from the library bintray apt repo
With the migration to a build environment based on docker and xenial, instructions for how to use the apt repo at bintray should be added to README and the manual

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
Fixes #248

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -55,6 +55,38 @@ major or minor **GitHub** release are designed to help developers to migrate fro
...
+Debian
+------
+
+For ``Debian`` based systems running **ubuntu xenial** ``deb`` packages are
+available in the project's **apt** repository at `bintray`_.
+
+To add the **apt** repository to your system follow the instructions given
+when clicking on the **[set up me]!** button.
+
+.. image:: docs/_static/bintray-setmeup.png
```
Comment (mathias-lang-sociomantic) [inline README.rst]: Y the underscore ?

File: docs/compiling.rst [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+    sudo apt install libdrizzle-redux (for versions on the v5.x branch)
+    sudo apt install libdrizzle-redux6 (for versions on the v6.x branch)
+
+.. _`bintray`: https://bintray.com/sociomantic-tsunami/libdrizzle-redux/libdrizzle-redux
+
+Redhat
+------
+Distribution packages are not available for ``Redhat`` based systems, but ``rpm``
+packages can be generated by running ``make rpm``.
+
```
Comment (mathias-lang-sociomantic) [inline docs/compiling.rst:32]: Y the duplication ? Can't you just link to the README if you want to provide the info here ?

File: README.rst (hunk 1 above)
Comment (andreas-bok-sociomantic) [reply README.rst]: > Y the underscore ?

What do you mean?

File: docs/compiling.rst (hunk 2 above)
Comment (andreas-bok-sociomantic) [reply docs/compiling.rst:32]: Yes I guess I could. But think it is better to have the pdf/html manual as self contained docs

File: README.rst (hunk 1 above)
Comment (mathias-lang-sociomantic) [reply README.rst]: `_static` ?

File: docs/compiling.rst (hunk 2 above)
Comment (mathias-lang-sociomantic) [reply docs/compiling.rst:32]: Then link from the README to `compiling`. A pure copy-paste is always a red flag, and prone to get out of sync at some point.

File: README.rst (hunk 1 above)
Comment (andreas-bok-sociomantic) [reply README.rst]: just the name of the directory. Legacy naming from when we forked

**Comment by andreas-bok-sociomantic** [comment]
Updated based on comments from @mathias-lang-sociomantic 
- Modified the section about installation in README.rst

**Review by mathias-lang-sociomantic — APPROVED** [review]

**Pull request merged**